ts/Core/Axis/Axis3D.ts

```diff
@@ -775,7 +775,7 @@ class Axis3D {
             }
             // If next label position is defined, then recalculate its position
             // basing on the perspective.
-            if (nextTick && (nextTick.label as any).xy) {
+            if (nextTick && nextTick.label && (nextTick.label as any).xy) {
```

Comment:
Does it mean we can get rid of `as any` castings above and below? :) Could you try it?